gui/browserv7/inc/ROOT/RBrowser.hxx

```diff
@@ -98,6 +98,7 @@ protected:
    std::string GetFileIcon(TString &name);
    std::string ProcessBrowserRequest(const std::string &msg);
    std::string ProcessDblClick(const std::string &path, const std::string &drawingOptions);
+   std::string getCurrentWorkingDirectory();
```

Comment:
Please use first capital letter for methods in c++ - this is ROOT naming convention

Reply:
Fixed in : 6404a15